tests/acceptance/features/webUICreateFilesFolders/createFolders.feature

```diff
@@ -53,14 +53,13 @@ Feature: create folders
     Then the page should be empty
   # Then the files table should be displayed
 
-  @skip @yetToImplement
+
   Scenario: Create a folder in a public share
     Given user "Alice" has created folder "simple-empty-folder"
+    And user "Alice" has shared folder "simple-empty-folder" with link with "read,create" permissions
     And the user has reloaded the current page of the webUI
```

Comment:
IMO, we don't need the page to be reloaded now. It was needed as the share was made using webUI.

Reply:
Yeah, that step seems useless here